### PR TITLE
Change predict inputs to table

### PR DESCRIPTION
**Pull request by LucasTrg**

### CODE REVIEW

File: webapp/src/app/query-runner/query-runner.component.html [hunk 1]
```diff
@@ -222,7 +222,31 @@
...
+                {{ field.name }}
+              </th>
+            </ng-container>
+          </tr>
+          <tr>
+            <ng-container *ngFor="let field of predict.fields">
+              <td mat-cell class="mat-cell cdk-cell">{{ field.value }}</td>
+            </ng-container>
+          </tr>
+          <!--<th mat-header-cell *matHeaderCellDef> {{ field.name }} </th>
```
Comment (ineiti) [inline webapp/src/app/query-runner/query-runner.component.html:270]: to be removed?

File: webapp/src/app/query-runner/query-runner.component.html [hunk 2]
```diff
@@ -222,7 +222,31 @@
...
+
+          <!--<tr mat-header-row *matHeaderRowDef="field. "></tr> 
+          <tr mat-row *matRowDef="let field; columns: columnsToDisplay"></tr>s-->
+        </table>
+
+        <button mat-raised-button color="primary" (click)="runPredictRequest()">
+          Predict
+        </button>
+
+        <!--<form
```
Comment (ineiti) [inline webapp/src/app/query-runner/query-runner.component.html:281]: Also remove?

File: webapp/src/app/query-runner/query-runner.component.html [hunk 3]
```diff
@@ -240,6 +264,8 @@
             </mat-form-field>
           </div>
 
+
```
Comment (ineiti) [inline webapp/src/app/query-runner/query-runner.component.html:299]: Is this from the prettier?

**Review by ineiti — DISMISSED** [review]
LGTM

File: webapp/src/app/query-runner/query-runner.component.css [hunk 4]
```diff
@@ -11,6 +11,20 @@ button {
...
+  overflow: hidden;
+  word-wrap: break-word;
+}
+
+.mat-cell {
+  padding: 5px;
+  align-items: center;
+  overflow: hidden;
+  word-wrap: break-word;
+}
```
Comment (tharvik) [inline webapp/src/app/query-runner/query-runner.component.css]: You can merge both definitions
```suggestion
.mat-cell, .mat-header-cell {
  padding: 5px;
  align-items: center;
  overflow: hidden;
  word-wrap: break-word;
}
```

File: webapp/src/app/query-runner/query-runner.component.html (hunk 2 above)
Comment (LucasTrg) [reply webapp/src/app/query-runner/query-runner.component.html:281]: Yep, sorry about this, I'll do my best to push more "polished" PR

**Pull request closed without merging**